src/controllers/upload.js

```
@@ -1,16 +1,52 @@
+
+    case 'subject':
+      result = await subjectService.updateSubject(id, {
+        appearance: { icon: imageUrl }
+      })
+      break
+
+    case 'user':
+      if (!email) {
+        return res.status(400).json({ error: 'Email is required for user updates' })
+      }
+      const user = await userService.getUserByEmail(email)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)